OnlineShop/OnlineShop.BLL/Services/Implementations/UserService.cs

```diff
@@ -1,8 +1,73 @@
-﻿using OnlineShop.BLL.Services.Interfaces;
+﻿using FluentValidation;
+using Mapster;
+using OnlineShop.BLL.DTO.Request;
+using OnlineShop.BLL.DTO.Response;
+using OnlineShop.BLL.Exceptions;
+using OnlineShop.BLL.Helpers;
+using OnlineShop.BLL.Mappers;
+using OnlineShop.BLL.Services.Interfaces;
+using OnlineShop.BLL.Validators;
+using OnlineShop.DAL.Entities.Implementations;
+using OnlineShop.DAL.Repositories.Interfaces;
 
 namespace OnlineShop.BLL.Services.Implementations;
 
-public class UserService:IUserService
+public class UserService : IUserService
 {
-    
+    private readonly IUserRepository _userRepository;
+
+    public UserService(IUserRepository userRepository)
+    {
+        MapsterConfig.Configure();
```

Comment:
The configuration can be moved outside of this class and registered where the di registration takes place